src/Equinor.ProCoSys.IPO.Command/InvitationCommands/EditInvitation/EditInvitationCommandHandler.cs

```diff
@@ -174,33 +174,33 @@ private async Task<List<CommPkg>> GetCommPkgScopeAsync(IList<string> commPkgNos,
 
         private async Task<List<BuilderParticipant>> UpdateParticipants(
             List<BuilderParticipant> participants,
-            IList<ParticipantsForCommand> participantsToUpdate,
+            IList<ParticipantsForEditCommand> participantsToUpdate,
             Invitation invitation)
         {
             var existingParticipants = invitation.Participants.ToList();
 
             var functionalRoleParticipants =
-                participantsToUpdate.Where(p => p.FunctionalRole != null).Select(p => p).ToList();
-            var functionalRoleParticipantIds = functionalRoleParticipants.Select(p => p.FunctionalRole.Id).ToList();
+                participantsToUpdate.Where(p => p.InvitedFunctionalRoleToEdit != null).Select(p => p).ToList();
```

Comment:
is the second select needed here?

Reply:
.Select(p => p) ?

Reply:
yes, it seems to just copy the list once more. Unless we need new references(Deep copy), I don't see the point.